docs/orchestration/flow_config/run_configs.md

```diff
@@ -189,6 +189,14 @@ Use a custom Kubernetes Job spec for this flow, stored in S3:
 flow.run_config = KubernetesRun(job_template_path="s3://bucket/path/to/spec.yaml")
 ```
 
+Always pull the newest image for the job run.  This should be used when using a custom docker image 
+with a tag other than `:latest`.  Doing so will set the kubernetes [imagePullPolicy](https://kubernetes.io/docs/concepts/containers/images/#updating-images)
+to `Always` on the job run.
+
+```python
+flow.run_config = KubernetesRun(always_pull_latest_image=True)
```

Comment:
The argument in the code is `always_pull_new_image`. I actually like `always_pull_latest_image` better. I would vote to update to `always_pull_latest_image` everywhere.

Reply:
Why not just expose `image_pull_policy` directly? Having a nonstandard kwarg will confuse those that are used to k8s, and we already make use of k8s kwarg names elsewhere.

Reply:
yeah fair point I like that better, +1 for exposing `image_pull_policy` directly

Reply:
yep, makes sense to me.  Do you have a preference on how that is exposed based on the rest of the repo? e.g.

1. type hint `Literal[Always, w/e]`
2. enum
3. just a string

Reply:
Just a string makes sense to me. We can check in the `KubernetesRun` constructor that the value is valid.